generators/client/templates/angular/src/main/webapp/app/layouts/profiles/_profile.service.ts

```diff
@@ -25,21 +25,29 @@ import { ProfileInfo } from './profile-info.model';
 @Injectable()
 export class ProfileService {
 
-    private profileInfoUrl = SERVER_API_URL + 'api/profile-info';
+    private infoUrl = SERVER_API_URL + 'management/info';
     private profileInfo: Promise<ProfileInfo>;
 
     constructor(private http: HttpClient) { }
 
     getProfileInfo(): Promise<ProfileInfo> {
         if (!this.profileInfo) {
-            this.profileInfo = this.http.get<ProfileInfo>(this.profileInfoUrl, { observe: 'response' })
+            this.profileInfo = this.http.get<ProfileInfo>(this.infoUrl, { observe: 'response' })
                 .map((res: HttpResponse<ProfileInfo>) => {
                     const data = res.body;
                     const pi = new ProfileInfo();
-                    pi.activeProfiles = data.activeProfiles;
-                    pi.ribbonEnv = data.ribbonEnv;
-                    pi.inProduction = data.activeProfiles.includes('prod') ;
-                    pi.swaggerEnabled = data.activeProfiles.includes('swagger');
+                    pi.activeProfiles = data['active-profiles'];
```

Comment:
its a bit inconsistent dont you think? all our json data uses camelcase for property keys, is this something we can change?

Reply:
Yes I can change this in the PR to the lib.